presto-parser/src/main/java/com/facebook/presto/sql/QueryUtil.java

```diff
@@ -181,6 +181,18 @@ public static Query simpleQuery(Select select, Relation from, Optional<Expressio
                 limit));
     }
 
+    public static Query singleValueQuery(String columnName, String value)
+    {
+        return query(new QuerySpecification(
```

Comment:
I think this is better than calling simpleQuery for two reasons. First, in simpleQuery we need to invent an alias for the 'table' which is required to be not null. Second, after a few calls in simpleQuery, it ended exactly the same here (i.e. creating a QuerySpecification with lots of empty Optionals). It is more efficient to just call this.